src/Game/Battleground.java

```diff
@@ -0,0 +1,134 @@
+package Game;
+
+import Game.Models.Field;
+import Game.Models.Map;
+import Game.Models.Player;
+import General.MB;
+import General.Shared.MBPanel;
+
+import java.awt.*;
+
+public class Battleground extends MBPanel {
+    /**
+     * The ratio of the battleground compared to the design size
+     */
+    public static float ratio = 0;
+    /**
+     * The offset to the top and to the left of the panel
+     */
+    public static int offset = 0;
+    /**
+     * The size of field
+     */
+    public static int size = 30;
```

Comment:
Variable could be named more precisely (e.g. fieldSize). Of course, the comment states the purpose of the variable, but elsewhere in the code, it's not directly clear. To look up the comment is rather cumbersome.